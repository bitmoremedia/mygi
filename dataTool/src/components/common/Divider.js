import React from 'react';
import styled from 'styled-components';

const StyledHr = styled.hr`
  border: 1px solid;
`;

const Divider = () => {
  return (
    <StyledHr />
  );
}

export default Divider;
