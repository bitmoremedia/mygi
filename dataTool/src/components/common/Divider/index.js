import React from 'react'
import styled from 'styled-components'

const StyledHr = styled.hr`
  border: 1px solid;
`
const Divider = () => <StyledHr />

export default Divider
