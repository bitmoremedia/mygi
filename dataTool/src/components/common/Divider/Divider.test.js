import React from 'react'
import { mount } from 'enzyme'
import renderer from 'react-test-renderer'

import Divider from './index'

test('component renders without error', () => {
  const wrapper = mount(
    <Divider />,
  )
  expect(wrapper.exists()).toBe(true)
})

// SNAPSHOT TESTS

test('component renders as expected', () => {
  const component = renderer.create(
    <Divider />,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
