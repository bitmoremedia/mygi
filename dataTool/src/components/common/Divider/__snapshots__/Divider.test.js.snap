exports[`test component renders as expected 1`] = `
<hr
  className="fTafup" />
`;
